engine/src/main/java/org/terasology/world/block/BlockUri.java

```diff
@@ -52,26 +57,36 @@ public BlockUri(String uri) throws BlockUriParseException {
                 throw new BlockUriParseException("Could not parse block uri: '" + uri + "'");
             }
             Name blockFamilyDefModule = new Name(split[0]);
+            ResourceUrn d;
+            Optional<ResourceUrn> s;
+            Name n;
```

Comment:
Please don't use one-letter abbreviations. This makes it harder to understand without any real benefit.